.github/workflows/assign_new_issue_to_project.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+name: Auto Assign Issues to Projects
+
+# yamllint disable-line rule:truthy
+on:
+  issues:
+    types: opened
```

Comment:
So it doesn't work for PRs, right?

Reply:
From what i can read in the docs, it doesn't. See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issues
there's a separate `pull_request` event.

If it turns out it does, we'll push a followup PR